cypress/integration/filters.test.ts

```diff
@@ -12,7 +21,7 @@ const defaultFilterConfigs = {
       field: 'vendor',
       title: 'Vendor',
       searchable: true,
-    },
+    } as Filter,
```

Comment:
Maybe just need to set `initial` to an empty array to avoid the casting?

```suggestion
    initial: []
    },
```

Reply:
then it will force other occurrences to have initial as required. I would want to avoid that

Reply:
Ok. I think we still have a better way to avoid that. Maybe we should have a global interface for the config but I'm ok to look into it in another time.